### ISSUE TITLE
Visualizing Robot with a recorded rosbag file

### ISSUE THREAD

**Issue by prof-concert** [ADVRHumanoids/concert_description#25]
![image](https://github.com/ADVRHumanoids/concert_description/assets/107113873/d92fbe00-5139-421a-a597-7c37624ad11a)
These are the rostopic I have 
![image](https://github.com/ADVRHumanoids/concert_description/assets/107113873/30fb90b1-e691-48bc-b134-326a0ed1d253)
and these are the rosnodes which are running

I have the concert description installed and sourced, instead of robot_description I have mentioned xbotcore/robot_description, still I am not able to visualize the robot in rviz.

**Comment by prof-concert** [comment]
Just one more info, I am using the concert_description docker container(running rviz) and profactors algorithm container(running algorithms and rosbag) with ROS_MASTER setup to visualize the robot model.

**Comment by EdoardoRomiti** [comment]
Hi @prof-concert , in order to visualizze the RobotModel on rviz you'll need:
- the `/tf` topic, which seem you have since you recorded it in the rosbag
- to set as a rosparam the Robot Description string. When you have xbot2 running this is uploaded as `xbotcore/robot_description`, but since now xbot2 is not running nobody has set this parameter. You can either set this by yourself or (better) **run the usual launch file without starting xbot2 and gazebo**. This can be done in this way for instance:
```
mon launch concert_gazebo concert.launch gazebo:=false xbot2:=false rviz:=true
```
This launch file will set the `robot_description_xbot` param that you can use in rviz to visualize the robot. 

In short, if you are running xbot2 use `xbotcore/robot_description`, if you are not use `robot_description_xbot` (e.g. when playing back a rosbag) .

**Comment by prof-concert** [comment]
@EdoardoRomiti Thanks a lot it worked. I was able to visualize the robot. closing this ticket.
